Bluewire.Common.GitWrapper/Model/RefHelper.cs

```diff
@@ -6,6 +6,8 @@ namespace Bluewire.Common.GitWrapper.Model
 {
     public static class RefHelper
     {
+        private static string RemoteHierarchy = "origin";

```

Comment:
This should be a parameter, possibly defaulted, but never assumed. While 'origin' is the default initial remote name by convention, it is not necessarily the correct one in all cases.